playbooks/db.yml

```diff
@@ -0,0 +1,20 @@
+---
+#
+# valet.sh | db
+#
+# @author: "Philipp Dittert"
+# @command: "db"
+# @description: "list, create, drop, reset, import, export a mysql database  | Usage: valet.sh db <action>"
+# @usage: "valet.sh db
+
```

Comment:
please add @help comment section for detailed help when calling
`valet.sh -h db`